homeassistant/components/airnow/sensor.py

```diff
@@ -58,6 +58,16 @@ class AirNowEntityDescription(SensorEntityDescription, AirNowEntityDescriptionMi
     """Describes Airnow sensor entity."""
 
 
+def station_extra_attrs(data: dict[str, Any]) -> dict[str, Any]:
+    """Process extra attributes for station location (if available)."""
+    if ATTR_API_STATION in data:
```

Comment:
so if `ATTR_API_STATION` is in there, the lat and long are there as well? Just to double check

Reply:
Yes, the station attributes all come as a group in the measurement data point from the API.  So as long as there is a PM2.5 measurement, all the station attributes will be there.